lldb/source/Target/Target.cpp

```diff
@@ -2401,35 +2400,38 @@ Target::GetScratchTypeSystemForLanguage(lldb::LanguageType language,
               errs->Flush();
             }
           }
-        }
```

Comment:
to save time for any other readers, this diff has code relayout which obscures the key part of this change: this closing brace (for `try_lock`) is moved to the end.